### ISSUE TITLE
Brak możliwości [ponownego] wystąpienia o zwrot prowizji

### ISSUE THREAD

**Issue by zaczy** [allegro/allegro-api#9992]
Mam problem z następującym scenariuszem (sandbox):

1. Poprzez API zwróciłem się o zwrot środków Klientowi ( [POST paiment/refunds](https://developer.allegro.pl/documentation#operation/initiateRefund) )
2. Jednocześnie wystąpiłem o zwrot prowizji ( [POST paiment/refund-claims](https://developer.allegro.pl/documentation#operation/createRefundApplication) )

Zwrot środków spotkał się z odmową (okazało się, że nie ma odpowiedniego salda na koncie Allegro Finanse)
W związku z powyższym prośba o zwrot prowizji również zakończyła się błędem

Po wpłynięciu na konto Allegro Finanse nowych środków prośba o zwrot na rzecz Klienta zakończyła się powodzeniem, ale przy ponowieniu prośby o zwrot prowizji otrzymuję odpowiedź o błędzie

```
{
	"errors": [
		{
			"code": "ClaimExistsException",
			"message": "Claim exists for line item id c1e1de21-766e-11ef-a645-a517c807654e.",
			"details": null,
			"path": null,
			"userMessage": "Refund application for the specified ID already exists."
		}
	]
} 
```
Przy odpytaniu API o istniejące wnioski o zwrot rzeczywiście widzę prośbę z pkt. 2 o statusie `REJECTED`

```
{
	"id": "cde11da8-9465-4e97-8db6-d5e24af5885b",
	"status": "REJECTED",
	"quantity": 3,
	"commission": {
		"amount": 2.89,
		"currency": "PLN"
	},
	"buyer": {
		"id": "107303758",
		"login": "Client:107303758"
	},
	"createdAt": "2024-09-19T11:08:08.446Z",
	"lineItem": {
		"id": "c1e1de21-766e-11ef-a645-a517c807654e",
		"quantity": 3,
		"boughtAt": "2024-09-19T10:07:53.375Z",
		"offer": {
			"id": "7775782511",
			"name": "DOLINA NOTECI Rafi junior jagniÄ™cina ĹĽurawina borĂłwka dla szczeniÄ…t 400g"
		}
	},
	"type": "MANUAL"
}
```

Czy to oznacza, że nie mogę złożyć wniosku o zwrot prowizji, jeśli już wcześniej próbowałem i z jakichś przyczyn się nie udało??

**Comment by MaciejFrackowiak** [comment]
Tak, dla każdego "lineItem.id" możesz utworzyć tylko jeden wniosek o rabat transakcyjny. Przy zwrocie płatności poprzez Allegro Finanse dokonujemy [automatycznego](https://help.allegro.com/sell/pl/a/jak-automatycznie-uzyskac-rabat-transakcyjny-zwrot-prowizji-9g0kEPbPghd) zwrotu prowizji.

**Comment by zaczy** [comment]
> Tak, dla każdego "lineItem.id" możesz utworzyć tylko jeden wniosek o rabat transakcyjny. 

[Dokumentacja API](https://developer.allegro.pl/documentation#operation/getRefundApplication) wymienia statusy `APPEALED` czy `REJECTED_AFTER_APPEAL`. Czy nie oznacza to, że odrzucony wniosek o rabat transakcyjny można "reklamować" po uzupełnieniu braków (np. po udanym zwrocie środków Klientowi)?

> Przy zwrocie płatności poprzez Allegro Finanse dokonujemy [automatycznego](https://help.allegro.com/sell/pl/a/jak-automatycznie-uzyskac-rabat-transakcyjny-zwrot-prowizji-9g0kEPbPghd) zwrotu prowizji.

Na Sandbox widzę, że dla zamówienia e688c491-7bd8-11ef-8144-999b443c0cca automatyczny zwrot nie nastąpił. Czy to oznacza, że ogólna historia konta nie spełnia jakichś warunków odnośnie automatycznego zwrotu?

**Comment by MaciejFrackowiak** [comment]
Można "reklamować" wyłącznie podczas bezpośredniego kontaktu z obsługą Allegro. Jednocześnie, na Sandbox nie działają wszystkie automatyczne mechanizmy, które są dostępne w produkcyjnym serwisie, ponieważ w przypadku płatności nie jest to możliwe do implementacji z uwagi na integrację z zewnętrznymi partnerami obsługującymi płatności.
